### DISCUSSION TITLE
Can't inovke treesitter's incremental selection with default keybindings <C-space>

### DISCUSSION THREAD

**Discussion by noidilin** [LazyVim/LazyVim#4044] [Q&A] [answered]
I can't invoke incremental selection with keybindings that related to <space> key. I have replaced <C-space> to <S-space>, yet still can't make it work. While mapping <C-space> to <C-x> works. In addition I can invoke incremental selection by searching in `<leader>sk` even when the <C-space> keybinding is unresponsive.

```lua
-- lua/plugins/treesitter.lua
return {
	{
		"nvim-treesitter/nvim-treesitter",
		opts = {
			incremental_selection = {
				enable = true,
				keymaps = {
					init_selection = "<S-space>",
					node_incremental = "<S-space>",
					scope_incremental = false,
					node_decremental = "<bs>",
				},
			},
		},
	},
}
```

This is my `init.lua`
```lua
require("config.lazy")
```

**Comment by folke** [comment, marked as answer]
That's a problem with your terminal

**Reply by noidilin** [reply]
Thanks for pointing me in the right direction. Everything is working as I expected now. Thank you!

I downloaded VS Code and tested the treesitter keybinding there, and it works smoothly. Therefore, it is probably an issue with my terminal, as all my devices and terminal emulators seem to have this weird behavior. However, I can't find the 'right' way to fix this problem, so I just let my terminal emulator send out a different key when shift+space is pressed. It works on all my devices.

**Reply by haphamdev** [reply]
I had to change keymap to `<leader><cr>` (Space Enter) instead
